### ISSUE TITLE
installing radon via pypi fails to install flake8_polyfill

### ISSUE THREAD

**Issue by gtristan** [rubik/radon#199]
In my gitlab CI job, I am installing radon with the following command:

```
pip3 install radon
```

When running radon, I get the following stack trace:

```
Traceback (most recent call last):
  File "/usr/local/bin/radon", line 10, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/dist-packages/radon/__init__.py", line 10, in main
    from radon.cli import program, log_error
  File "/usr/local/lib/python3.7/dist-packages/radon/cli/__init__.py", line 10, in <module>
    import radon.complexity as cc_mod
  File "/usr/local/lib/python3.7/dist-packages/radon/complexity.py", line 7, in <module>
    from flake8_polyfill import options
ModuleNotFoundError: No module named 'flake8_polyfill'
```

This started happening some time over the last couple days.

**Comment by sr-verde** [comment]
I have experienced the same issue.
Installing `flake8_polyfill` manually is a work around.

**Comment by cblegare** [comment]
If `flake8_polyfill` is optional, it should be hidden behind a conditional import

here, for instance

https://github.com/rubik/radon/blob/master/radon/complexity.py#L7

Also, if this changed did not trigger a error during the travis pipeline (false negative), probably something is wrong in dependency management from within the pipeline.

**Comment by sr-verde** [comment]
For clarification, I don't use travis but I use xenon executed by a gitlab runner.

**Comment by sr-verde** [comment]
Seems to be fixed.

**Comment by tribals** [comment]
Please, see issue #189 an PR #200.

**Comment by rubik** [comment]
@gtristan @sr-verde @cblegare @tribals Sorry for the installation issue. It's been fixed in version 4.3.2 which I just uploaded.
